default.yml

```diff
@@ -68,3 +37,59 @@ Style/PercentLiteralDelimiters:
 
 Style/RegexpLiteral:
   Enabled: false
+
+#################### Naming ################################
+
+Naming/FileName:
+  Enabled: false
+
+#################### Metrics ###############################
+
+Metrics/AbcSize:
+  Max: 22.5
+
+Metrics/BlockLength:
+  Exclude:
+  - config/routes.rb
+  - app/views/api/**/*
+
+Metrics/ClassLength:
+  Max: 150
+
+Metrics/CyclomaticComplexity:
+  Max: 9
+
+Metrics/LineLength:
+  Max: 110
+  # The IgnoreCopDirectives option causes the LineLength rule to ignore cop
+  # directives like '# rubocop: enable ...' when calculating a line's length.
+  IgnoreCopDirectives: true
+  Exclude:
+    - config/routes.rb # Annotations run long.
+
+Metrics/MethodLength:
```

Comment:
I think this is still too low.

Reply:
Well this is a interesting discussion to have, the way I see it if we go up to 20 lines on MethodLength we would also need to update CyclomaticComplexity, AbcSize, PerceivedComplexity accordingly, because as the method grows in length is most likely that it will grow in complexity.